solver/llbsolver/solver.go

```diff
@@ -125,13 +134,16 @@ func (s *Solver) Bridge(b solver.Builder) frontend.FrontendLLBBridge {
 	return s.bridge(b)
 }
 
-func (s *Solver) Solve(ctx context.Context, id string, sessionID string, req frontend.SolveRequest, exp ExporterRequest, ent []entitlements.Entitlement, post []Processor) (_ *client.SolveResponse, err error) {
-	j, err := s.solver.NewJob(id)
-	if err != nil {
-		return nil, err
-	}
+func (s *Solver) recordBuildHistory(ctx context.Context, id string, req frontend.SolveRequest, j *solver.Job) (func(*Result, exporter.DescriptorReference, error) error, error) {
```

Comment:
We should have a tracking issue for SBOM support with build history if we're not doing it in this PR.